docker-for-windows/troubleshoot.md

```diff
@@ -81,6 +81,27 @@ Diagnostics Bundle: C:\Users\User\AppData\Local\Temp\CD6CF862-9CBD-4007-9C2F-5FB
 Diagnostics ID:     CD6CF862-9CBD-4007-9C2F-5FBE0572BBC2/20180720152545 (uploaded)
 ```
 
+### Experimental self-diagnose tool
+
+Docker Desktop 3.6.0 contains an experimental "self-diagnose" tool which helps identify some common
+problems. To run it: first locate `com.docker.diagnose.exe`, usually installed in `C:\Program
+Files\Docker\Docker\resources\com.docker.diagnose.exe`.
+
+To run the self-diagnose tool in Powershell:
+
+```powershell
+  PS C:\> & "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check
```

Comment:
Looks like there's some indenting here that we should remove (not urgent)

```suggestion
PS C:\> & "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check
```

Reply:
Thanks. Will fix in a follow-up PR

Reply:
Fixed in https://github.com/docker/docker.github.io/pull/13332